examples/seismic/acoustic/wavesolver.py

```diff
@@ -38,6 +40,11 @@ def __init__(self, model, source, receiver,
         # Cache compiler options
         self._kwargs = kwargs
 
+        if ~isinstance(model.m, DenseData):
```

Comment:
Habit :)
This  is gonna go away anyway tto swich to `compare_type(self.model.m, m)`, just need to figure out how to access Class type of m.